### PR TITLE
Fix starting isolate directly from CoreJIT snapshot.

### PR DESCRIPTION
**Pull request by sbaranov**

### CODE REVIEW

File: shell/common/isolate_configuration.cc [hunk 1]
```diff
@@ -37,9 +37,6 @@ class PrecompiledIsolateConfiguration final : public IsolateConfiguration {
```
Comment (rmacnak-google) [inline shell/common/isolate_configuration.cc]: This class should be renamed to something like AppSnapshotIsolateConfiguration

File: shell/platform/android/platform_view_android_jni.cc [hunk 2]
```diff
@@ -190,7 +190,8 @@ std::unique_ptr<IsolateConfiguration> CreateIsolateConfiguration(
     return script;
   }
 
-  return nullptr;
+  // This happens when starting isolate directly from CoreJIT snapshot.
+  return IsolateConfiguration::CreateForPrecompiledCode();
```
Comment (rmacnak-google) [inline shell/platform/android/platform_view_android_jni.cc]: Similar rename.

File: shell/common/isolate_configuration.cc (hunk 1 above)
Comment (sbaranov) [reply shell/common/isolate_configuration.cc]: Done.

File: shell/platform/android/platform_view_android_jni.cc (hunk 2 above)
Comment (sbaranov) [reply shell/platform/android/platform_view_android_jni.cc]: Done.

**Pull request merged**